lib/matplotlib/tests/test_path.py

```diff
@@ -49,25 +49,26 @@ def test_contains_points_negative_radius():
     np.testing.assert_equal(result, [True, False, False])
 
 
-_test_paths = [
+_ExampleCurve = namedtuple('ExampleCurve', ['path', 'extents', 'area'])
+_test_curves = [
     # interior extrema determine extents and degenerate derivative
-    Path([[0, 0], [1, 0], [1, 1], [0, 1]],
-           [Path.MOVETO, Path.CURVE4, Path.CURVE4, Path.CURVE4]),
-    # a quadratic curve
-    Path([[0, 0], [0, 1], [1, 0]], [Path.MOVETO, Path.CURVE3, Path.CURVE3]),
+    _ExampleCurve(Path([[0, 0], [1, 0], [1, 1], [0, 1]],
+                       [Path.MOVETO, Path.CURVE4, Path.CURVE4, Path.CURVE4]),
+                  extents=(0., 0., 0.75, 1.), area=0.6),
+    # a quadratic curve, clockwise
+    _ExampleCurve(Path([[0, 0], [0, 1], [1, 0]], [Path.MOVETO, Path.CURVE3,
```

Comment:
Minor thing, but I'd still wrap this at the end of the vertices, so you aren't breaking up an argument.